internal/ui/settings.go

```diff
@@ -183,6 +194,11 @@ func (d *deskSettings) load() {
 	if moduleNames != "" {
 		d.moduleNames = strings.Split(moduleNames, "|")
 	}
+
+	d.clockFormatting = fyne.CurrentApp().Preferences().StringWithFallback("clockformatting", "12h")
+	if d.clockFormatting != "12h" && d.clockFormatting != "24h" {
```

Comment:
Is this code block needed?
In widgetpanel.go there is a `if == "12h"` - if that were reversed to "24h" then this would not be needed I think... anything other than "24h" will show 12, which is the default

Reply:
Good point. Should probably be possible to remove it.